app/__init__.py

```diff
@@ -29,34 +29,47 @@ def do_login(user):
     session[CURR_USER_KEY] = user.id
 
 
-@app.route('/register', methods=["GET", "POST"])
-def register_user():
-    """Handles user signup."""
-    form = RegisterUser()
```

Comment:
@azaria-dedmon why are we removing this functionality.

Reply:
I am only including routes that are related to login/logout functionality.

Reply:
@azaria-dedmon Note that a PR is essentially an edit of what already exists in the master branch. When we add or remove something in a PR and it is merged, those changes will take effect in the master branch. And since this PR is only meant to add login/logout functionality, we should not be removing the registration functionality.